.gitignore

```diff
@@ -34,6 +34,3 @@ tpch/data/*
 
 # MacOS
 .DS_Store
-
-# IntelliJ IDEA
-.idea/
```

Comment:
@Riik why was this removed? I would consider this the same as `.vscode` but then for IntelliJ and Pycharm. It should not be pushed.

Reply:
I agree it would be easier to have .idea in gitginore as well, but figured it was easier to keep that separate from this PR since it was just about the bug ticket

Reply:
I will remove the README clean up then too :)